Trainee/Reviews/ViewReviews/ViewReviews.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import axios from "../../../../api/axios";
+import { faStar } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FaStar } from "react-icons/fa";
+import "./ViewReviews.css";
+
+export default function ViewReviews({ trainerName }) {
+  const [reviews, setReviews] = useState([]);
+  console.log("review", reviews);
+  useEffect(() => {
+    async function getReviews() {
+      const response = await axios.get("/review");
+      setReviews(
+        response.data.filter((review) => {
+          return review.trainerName === trainerName;
+        })
+      );
+    }
+    getReviews();
+  }, []);
+  function handleRatings(rating) {
+    while (rating > 0) {
```

Comment:
why is this a while statement? should an IF?

Reply:
I have removed that section